templates/ingress.yaml

```diff
@@ -68,7 +72,7 @@ spec:
 {{- if and .Values.ingress.enabled (eq .Values.airflow.executor "CeleryExecutor") }}
 ---
 kind: Ingress
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Should this have the same backcompat for <1.19 (this is also an ingress)?

Reply:
[<1.19 is already deprecated in all cloud providers.](https://github.com/astronomer/ops-stuff/blob/726f67f1f4/docs/k8s-version-table.md)

Reply:
Cool, so then we don't need backcompat on the other ingress, right?

Reply:
That's right. IMHO we can drop everything <1.19 including CI tests for those versions.

Reply:
Looks like we already dropped <1.19 from CI 👍